andela/Ah-frontend-guardians
# #163935489 Update the README file

**What does this PR do?**
This PR Updates the README.md file with badges and how the app can be tested locally

**Description of Task to be completed?**
- Add CircleCI badge
- Add TravisCI badge
- Add Hound badge
- Updated CircleCI.yml file to get test coverage report from Codeclimate
- Add the Codeclimate Test Coverage badge

**How should this be tested?**
- Go to this repository [Here](https://github.com/andela/Ah-frontend-guardians/tree/develop) and go to the develop branch
- To see the badges if not merged yet,  go to this [Link](https://github.com/andela/Ah-frontend-guardians/tree/ch-update-README-163935489)
- `Test Coverage` badge shows **unknown** on the `ch-update-README-163935489` branch because it points to the develop branch

**Relevant pivotal tracker stories**
[#163935489](https://www.pivotaltracker.com/story/show/163935489)

<img width="998" alt="screen shot 2019-02-13 at 5 11 02 pm" src="https://user-images.githubusercontent.com/40206038/52718234-0e2d9d80-2fb4-11e9-9d99-9a94d68a40d4.png">
